app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -467,6 +467,23 @@
   <dimen name="hints_and_solution_recycler_view_padding_top">0dp</dimen>
   <dimen name="hints_and_solution_recycler_view_padding_bottom">112dp</dimen>
 
+  <!-- Hints summary  -->
+  <dimen name="hints_summary_constraint_layout_layout_margin_start">128dp</dimen>
+  <dimen name="hints_summary_constraint_layout_layout_margin_end">128dp</dimen>
+  <dimen name="hints_summary_constraint_layout_layout_margin_bottom">4dp</dimen>
+
+  <dimen name="hint_list_container_layout_margin_start">128dp</dimen>
```

Comment:
Name for every dimension should start from name of activity/fragment. So, here and everywhere else start the name of dimension with hints_summary as you have done for previous dimensions. Here, change the current name `hint_list_container_layout_margin_start` to `hints_summary_hint_list_container_layout_margin_start`